### PR TITLE
return next_natural_triggerls

### LINKED ISSUE #116: missing next_natural_trigger value in response
`curl http://<host>/api/v2/workflows/<cid>/<wfid>/state`

`next_natural_trigger` in response is always `null`

### PR DESCRIPTION
**Pull request by honnix**
fix #116

### CODE REVIEW

**Comment by fabriziodemaria** [comment]
We should update tests in `WorkflowResourceTest` as well

**Comment by rouzwawi** [comment]
This should return both the trigger and offset triggers.

File: styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java [hunk 1]
```diff
@@ -421,6 +421,7 @@ public WorkflowState workflowState(WorkflowId workflowId) throws IOException {
     WorkflowState.Builder builder = WorkflowState.builder().enabled(enabled(workflowId));
     getDockerImage(workflowId).ifPresent(builder::dockerImage);
     getCommitSha(workflowId).ifPresent(builder::commitSha);
+    getNextNaturalTrigger(workflowId).ifPresent(builder::nextNaturalTrigger);
```
Comment (rouzwawi) [inline styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java]: Can we refactor this method to not fetch the workflow once for each property?

**Comment by honnix** [comment]
@rouzwawi PTAL?

**Comment by rouzwawi** [comment]
LGTM!

**Pull request merged**